cocos/spine/lib/spine-core.d.ts

```diff
@@ -267,8 +267,6 @@ declare namespace spine {
         setFrame(frameIndex: number, time: number, rotateMix: number, translateMix: number): void;
         apply(skeleton: Skeleton, lastTime: number, time: number, firedEvents: Array<Event>, alpha: number, blend: MixBlend, direction: MixDirection): void;
     }
-}
-declare namespace spine {
```

Comment:
Pay attention to API export result, whether spine is treated as general namespace for all spine core APIs